lib/config/integrations/rails/railtie.rb

```diff
@@ -23,11 +23,13 @@ def preload
         # Development environment should reload settings on every request
         if ::Rails.env.development?
           initializer :config_reload_on_development do
-            ActionController::Base.class_eval do
-              if ::Rails::VERSION::MAJOR >= 4
-                prepend_before_action { ::Config.reload! }
-              else
-                prepend_before_filter { ::Config.reload! }
+            ActiveSupport.on_load :action_controller do
```

Comment:
Wouldnt https://github.com/railsconfig/config/pull/250/files#diff-e1b40370ec751eed3246e103922adc0eR26 be enough (without `ActionController::Base.class_eval`)?